src/pages/public_registration/PublicPublicationContext.tsx

```diff
@@ -22,13 +22,11 @@ export const PublicJournalContent = ({ date, publicationContext }: PublicJournal
   return title ? (
     <>
       {url ? (
-        <Typography variant="subtitle2" component={Link} href={url} target="_blank" rel="noopener noreferrer">
+        <Typography variant="overline" component={Link} href={url} target="_blank" rel="noopener noreferrer">
```

Comment:
Jeg forsto det som at vi skulle ha "Tidsskrift" som `overline` og tittelen under, likt som for utgiver/serie?

Reply:
åja, jeg forstod det ikke sånn 😅 hører med Kine igjen